ntbs-service/wwwroot/source/Components/FilteredDropdown.ts

```diff
@@ -0,0 +1,122 @@
+import Vue from "vue";
+import { getHeaders } from "../helpers";
+import axios from "axios";
+
+type OptionValue = {
+    value: string,
+    text: string,
+    group: string
+};
+
+const FilteredDropdown = Vue.extend({
```

Comment:
I feel like this component could use a short comment above to explain how to use it when wiring it up, what do you think?